content/guides/accessibility-for-teams/front-end-development.md

```diff
@@ -0,0 +1,232 @@
+  orginal: https://youtu.be/cOmehxAU_4s?t=3m25s
+ -->
+{{< youtube cOmehxAU_4s >}}
+
+- [Download NVDA screen reader (NV Access)](https://www.nvaccess.org/download/)
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=128%2C14&currentsidebar=%23col_overview#content-structure-separation-programmatic)
+
+## Headings
+
+{{< ring title="Why it's important" >}}
+Are you using accurate headings on your page?
+
+- Lalit is blind and uses a screen reader to navigate the web. She hears an outline of the page's main ideas, then backtracks to read the parts she's most interested in.
+{{< /ring >}}
+
+
+### Steps to take
+- Headings briefly describe the section it introduces. Headings represent an outline of the content.
+- Use `h1`–`h6` to define your section headings, where `h1` is the highest section level and `h6` is the lowest.
+- Avoid skipping heading levels: Always start with `h1`, use `h2` next, and so on.
+- Use only one `h1` per page for the page title.
+
+While HTML5 allows you to reset headings to h1 on new section elements, some screen reader users will have difficulty discerning the structure of pages with multiple h1s. For this reason, it’s best to include only one h1 per page.
+
+### Resources
+
+**Video tutorial: Using headings**
+
+{{< youtube ZHWcs5d9IqA >}}
+
+**Video tutorial: How I do an accessibility check (page structure)**
+
+<!-- 
+  TODO: need to refactor hugo shortcode to use timestamp in youtube urtl
+  orginal: https://youtu.be/cOmehxAU_4s?t=6m21s
+ -->
+
+{{< youtube cOmehxAU_4s >}}
+
+- [2.4.6 Headings and Labels](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=128%2C14&currentsidebar=%23col_overview#navigation-mechanisms-descriptive)
+
+
+## Page structure
+
+{{< ring title="Why it's important" >}}
+Are you using semantic elements and roles?
+
+- Carlos is low-sighted and navigates pages by jumping to the page section he wants to get to.
+{{< /ring >}}
+
+### Steps to take
+- Use sectioning elements to create a broad outline of your page content; examples of these elements include `header`, `nav`, `main`, and `footer`. Use content sectioning elements like `section`, `article`, and `aside` to organize the document content into logical pieces.
+- Add `role="banner"` to your masthead and `role="contentinfo"` to your page footer once per page to define landmark elements.
+
+### Resources
+
+**Video tutorial: Landmarks**
+
+<!-- 
+  TODO: need to refactor hugo shortcode to use timestamp in youtube urtl
+  orginal: https://www.youtube.com/watch?time_continue=12&v=bww3IaktlRY
+ -->
+
+{{< youtube bww3IaktlRY >}}
+
+- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=14%2C128&currentsidebar=%23col_overview#content-structure-separation-programmatic)
+- [2.4.1 Bypass Blocks](https://www.w3.org/WAI/WCAG20/quickref/?showtechniques=14%2C128&currentsidebar=%23col_overview#navigation-mechanisms-skip)
+
+## Images
```

Comment:
In the build I'm seeing "Images" in 2 headings: 
![Image 2-20-24 at 3 24 PM](https://github.com/GSA/digitalgov.gov/assets/96838068/10b17b69-7ccc-44a5-84cc-6294143f2a2c)

Reply:
This looks like a caching issue, trying clearing the browser cache.